gbpservice/tests/contrib/devstack/local-nfp.conf

```diff
@@ -46,13 +49,33 @@ enable_service q-agt
 enable_service q-dhcp
 enable_service q-l3
 enable_service q-fwaas
+[[ $ENABLE_NFP = True ]] && [[ $NFP_DEVSTACK_MODE = advanced ]] && enable_service neutron-vpnaas
 enable_service q-lbaas
 enable_service q-meta
 enable_service neutron
 enable_service group-policy
 disable_service tempest
 ENABLED_SERVICES+=,heat,h-api,h-api-cfn,h-api-cw,h-eng
-enable_service orchestrator,proxy,proxy_agent,base_configurator
+
+if [[ $ENABLE_NFP = True ]]; then
+    # NFP services
+    enable_service nfp_orchestrator
+    enable_service nfp_proxy
+    enable_service nfp_proxy_agent
+    [[ $NFP_DEVSTACK_MODE = base ]] && enable_service nfp_base_configurator
+    [[ $NFP_DEVSTACK_MODE != base ]] && enable_service nfp_config_orchestrator
+    [[ $NFP_DEVSTACK_MODE = advanced ]] && enable_plugin neutron-vpnaas https://git.openstack.org/openstack/neutron-vpnaas stable/mitaka
```

Comment:
Use [[ $NFP_DEVSTACK_MODE = advanced ]] instead of [[ $NFP_DEVSTACK_MODE != base ]]

Also, why "enable_plugin neutron-vpnaas" ? We already did "[[ $ENABLE_NFP = True ]] && [[ $NFP_DEVSTACK_MODE = advanced ]] && enable_service neutron-vpnaas" above right.